src/lib/dialog/dialog-ref.ts

```diff
@@ -30,11 +29,14 @@ export class MdDialogRef<T> {
   /** Whether the user is allowed to close the dialog. */
   disableClose = this._containerInstance._config.disableClose;
 
+  /** Subject for notifying the user that the dialog has finished opening. */
+  private _afterOpen = new Subject<void>();
+
   /** Subject for notifying the user that the dialog has finished closing. */
-  private _afterClosed: Subject<any> = new Subject();
+  private _afterClosed = new Subject<any>();
```

Comment:
These subjects should be `Subject<void>`.

Reply:
Don't they carry the close value of the dialog?